src/utils/request/fetchAllRequest.js

```diff
@@ -0,0 +1,12 @@
+import singleRequest from './singleRequest';
+
+const fetchAllRequest = async (url, amountOfRequests, fetchOptions = {}) => {
+  const data = await Promise.all(
+    [...new Array(amountOfRequests - 1)] // -1 because we already did a firstRequest
```

Comment:
No point in destructuring into a new array here, just do Promise.all(new Array(...)...)

Reply:
I was confused by that as well. A simple

```js
new Array(amountOfRequests - 1)
  .map((_, i) => singleRequest(url, { ...fetchOptions, page: { i + 2 } })),
```

should do it, right?

Reply:
I wasn't able to remove the spread, no idea why not.

Reply:
Ah yes, looks like it's not possible to map over an array that way because it only initializes the `length`, my bad

Reply:
Luckily we have tests that let us test out things 😅